source/controlTypes/deprecatedAliases.py

```diff
@@ -199,6 +199,7 @@
 STATE_PRESSED = State.PRESSED
 STATE_CHECKED = State.CHECKED
 STATE_HALFCHECKED = State.HALFCHECKED
+STATE_HALFPRESSED = State.HALFPRESSED

```

Comment:
As per the warning in this file, this module is for deprecated code. please do not add to it